02-Logging/01-Logging-to-a-GSD-file.ipynb

```diff
@@ -230,7 +230,7 @@
     {
```

Comment:
<p>I think this text block should be in a smaller, not bolded font. While reading this, I feel like the notebook is yelling at me.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/95/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
I do not know why ReviewNB renders the notebook this way. It is not present in the source:
See https://github.com/glotzerlab/hoomd-examples/blob/gsd-flush/02-Logging/01-Logging-to-a-GSD-file.ipynb
and 
```
 {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Flush the GSD buffer so it is readable in the next code cell. In typical workflows, you will run separate simulation and analysis scripts and the buffered writes will automatically flush when your simulation script exits."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {
    "tags": []
   },
   "outputs": [],
   "source": [
    "gsd_writer.flush()"
   ]
  },
```